### ISSUE TITLE
Use custom desktop names in Task View

### ISSUE THREAD

**Issue by msevcenko** [sdias/win-10-virtual-desktop-enhancer#107]
This is good tool to organize work when working on many projects in parallel.

It would be much more helpful if you could switch to correct task from the Task View (shown after pressing Win-Tab). This view, however, displays default generic desktop names (Desktop 1, Desktop 2), which is not very helpful.

**Comment by GioBonvi** [comment]
Hi @msevcenko,
Windows does not actually let you rename the desktops the way you are asking for: the names you can assign in the script are only used by the script itself (in the popups and in the hover text).

You might want to follow #55 to stay updated on this.

Closing this as it is a duplicate of #55.

**Comment by msevcenko** [comment]
Hi, ok thanks for the info.
